cpp/src/arrow/io/buffered_test.cc

```diff
@@ -293,6 +293,15 @@ TEST_F(TestBufferedOutputStream, Tell) {
 
   ASSERT_OK(buffered_->Close());
 
+  // write long bytes after raw_pos is cached
+  const std::string basic_string = "1234568790";
+  OpenBuffered(/*buffer_size=*/3, false);
```

Comment:
Last nit comment :)
 `false` looks not necessary. If you think it's important, please also add `/*append=*/false`, boolean variables are hard to reason from caller side.

Reply:
done